.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*.php]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = tab
+indent_size = 4
```

Comment:
This avoid tabs/spaces conflicts in files out of the box

![image](https://user-images.githubusercontent.com/924196/107156050-4cc87980-697c-11eb-82e0-9a00539ab8ca.png)

Reply:
CS is in `/build-cs` and runs only on PHP 7.4 afaik. PHP_CodeSniffer is probably not yet compatible with PHP 8.

Reply:
Yes :/ 
https://github.com/phpstan/phpdoc-parser/pull/57#issuecomment-774729574

Reply:
This package is blocker for PHP 8: https://github.com/consistence/coding-standard/blob/4b4b199b14071c26d6b8064a19a4227c1700a000/composer.json#L14

Reply:
It seems @vasekPurchart is not maintaining that packages for a while https://github.com/consistence/coding-standard/pull/65

Reply:
Resolved in standalone PR https://github.com/phpstan/phpdoc-parser/pull/58